### DISCUSSION TITLE
Component not rendering when passing attributes with new short attribute syntax

### DISCUSSION THREAD

**Discussion by devsquad-victor-lima** [livewire/livewire#5325] [7. Bugs (archived)]
**The Issue**
Given the following component call:

`<livewire:some-component :$foo :$bar />`

It won't render properly. Instead, it "renders" like:

`<livewire:some-component :$foo="" :$bar="">
</livewire:some-component>`

Am I doing something wrong?

**Comment by ramdoni** [comment]
try this

`<livewire:some-component :foo="$foo" :bar="$bar">`
 or 
`@livewire('some-component', ['foo'=>$foo,'bar'=>$bar])`

you can read this docs [https://laravel-livewire.com/docs/2.x/rendering-components](url)

**Comment by joshhanley** [comment]
@devsquad-victor-lima while support was added for that in blade, support is yet to be added to Livewire for the `<livewire:... />` directive.

So currently this won't work.

Hope this helps!

**Comment by kirkbushell** [comment]
Short attribute syntax simply will not work while you're using livewire, regardless of whether or not it's a Livewire component. And unfortunately, there doesn't appear to be anyway around this. I was stuck on this for hours, trying to figure out why perfectly fine code wouldn't render when in a livewire environment (I wrote tests and everything functioned fine).

Instead of this:

```html
<input :$type
```

Or even:

```html
<input :type="$type"
```

You have to use completely verbose syntax:

```html
<input type="{{ $type }}"
```

It's a minor inconvenience, but quite annoying. I'm not sure why Livewire does this and tbh, quite surprised it's still here in the codebase, been a while now.

**Reply by lrljoe** [reply]
If you're not using a component, then I'd expect it to behave as you've written?   The idea is that you use blade components to abstract from raw HTML to help you minimise re-writing of code needlessly throughout your project.

To my mind, there is zero purpose to adding support for attributes to raw HTML, unless I'm really missing something here?

Something like the below **does** work:
```
<x-forms.input-text :labelVariabe="$label" />
```
Where your "views/components/forms/input-text.blade.php" contains the relevant props and code to figure out what to do with that property.

Really not sure why you'd think that using "<input />" should support this?

Passing into a Livewire Component like below also works, so long as you've setup your properties correctly
```
<livewire:some-component :foo="$foo"  :bar="$bar"> </livewire:some-component>
```
